client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -134,14 +134,18 @@ export const AppSchemaForm = ({ app }) => {
     const matchingHost = Object.keys(state.allocations.hosts).find(
       host => app.resource === host || app.resource.endsWith(`.${host}`)
     );
+    const host = state.allocations.hosts[state.systems.storage.defaultHost];
+    const hasCorral =
+      host === 'cloud.corral.tacc.utexas.edu' || 'data.tacc.utexas.edu';
```

Comment:
I think as it stands, this is just assigning `hasCorral` to `data.tacc.utexas.edu` if the first phrase is falsy. 
```suggestion
      host in ['cloud.corral.tacc.utexas.edu', 'data.tacc.utexas.edu'];
```
```suggestion
      host === 'cloud.corral.tacc.utexas.edu' || 'data.tacc.utexas.edu';
```